scalahost/sbt/src/main/scala/scala/meta/scalahost/sbt/ScalahostSbtPlugin.scala

```diff
@@ -22,15 +40,21 @@ object ScalahostSbtPlugin extends AutoPlugin {
 
   lazy val scalahostAllSettings =
     scalahostBaseSettings ++
-      scalahostInjectCompilerPluginSettings ++
+      inTestAndCompile(scalahostConfigSettings) ++
+      scalahostInjectScalacOptions ++
+      scalahostInjectDependency ++
       scalahostHasMirrorSettings
 
   lazy val scalahostBaseSettings: Seq[Def.Setting[_]] = Def.settings(
     ivyConfigurations += Scalameta,
+    scalametaSourcepath := baseDirectory.in(ThisBuild).value,
     resolvers += Resolver.bintrayRepo("scalameta", "maven")
   )
+  lazy val scalahostConfigSettings: Seq[Def.Setting[_]] = Def.settings(
+    scalametaSemanticdb := ScalametaSemanticdb.Fat
```

Comment:
It's conventional to prefix all plugin keys by the plugin name, this makes it easier to discover what settings are provided by a plugin, for example via `scalameta<TAB>` in the sbt shell. I'm open to make the syntax `scalametaSemanticdb := Semanticdb.Fat`, would that be better?